test/unit/preact/test-base-element.js

```diff
@@ -531,6 +539,25 @@ describes.realWin('PreactBaseElement', {amp: true}, (env) => {
       await waitFor(() => component.callCount > 1, 'component re-rendered');
       expect(component).to.be.calledTwice;
     });
+
+    it('clones children (without descendant) as vnodes into prop', async () => {
+      expect(
+        component.withArgs(
+          env.sandbox.match({
+            cloned: [
+              env.sandbox.match({
+                type: 'DIV',
+                props: {cloned: '', id: 'cloned1'},
```

Comment:
For `{cloned: ''}`, component functions expect this as a boolean `{cloned: true}`.

I'd like to set this like:

```js
props[name] = value === '' ? true : value;
```

but I'm not sure if it's weird to special-case this.